packages/react-pose/src/components/PoseElement/types.ts

```diff
@@ -23,6 +23,7 @@ export type PoseElementProps = {
   _pose?: CurrentPose;
   initialPose?: CurrentPose;
   withParent?: boolean;
+  onPoseComplete?: (pose: CurrentPose) => any;
```

Comment:
This would be `pose: string`

Reply:
if we change it, currently the typing is correct (right?) - `CurrentPose` might be an array of strings

Reply:
I meant in the event of changing the above, but yeah this is fine